### PR TITLE
Update domain

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: src/main/java/io/gingersnap_project/GreetingResource.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.gingesnap;
+package io.gingersnap_project;
```
Comment (pruivo) [inline src/main/java/io/gingersnap_project/GreetingResource.java:1]: as a package name, I would rather have just a single word: `io.gingersnapproject`
like we have in Infinispan project, for example, the global state package: `org.infinispan.globalstate`

Comment (ryanemerson) [reply src/main/java/io/gingersnap_project/GreetingResource.java:1]: IMO that's harder to read. We also don't own `gingersnapproject.io/`, which is why we can't just do `io.gingersnap`

Comment (pruivo) [reply src/main/java/io/gingersnap_project/GreetingResource.java:1]: is it a legal thing?

Comment (ryanemerson) [reply src/main/java/io/gingersnap_project/GreetingResource.java:1]: I don't know if they go as far as checking imports :shrug:, @tristantarrant probably knows.

I was coming from the angle of best practice

Comment (ryanemerson) [reply src/main/java/io/gingersnap_project/GreetingResource.java:1]: I spoke with Tristan and apparently OSSRH releases can be a pain if you don't own the domain. Let's stick with `io.gingersnap_project` for now and we can revisit later if we feel it's an issue.

**Pull request merged**